leaflet-providers.js

```diff
@@ -132,19 +138,61 @@
 				}
 			}
 		},
+		OpenInfraMap: {
+			url: 'https://tiles-{s}.openinframap.org/{variant}/{z}/{x}/{y}.png',
+			options: {
+				maxZoom: 18,
+				attribution: 
+					'{attribution.OpenStreetMap}, <a href="http://www.openinframap.org/about.html">About OpenInfraMap</a>'
+			},
+			variants: {
+				  Power:       'power' ,
+				  Telecom:     'telecoms' ,
+				  Petroleum:   'petroleum' ,
+				  Water:       'water'
+			}
+		},
 		OpenSeaMap: {
 			url: 'https://tiles.openseamap.org/seamark/{z}/{x}/{y}.png',
 			options: {
 				attribution: 'Map data: &copy; <a href="http://www.openseamap.org">OpenSeaMap</a> contributors'
 			}
 		},
+		OpenPtMap: {
+			maxZoom: 17,
```

Comment:
I think this one has to go into `options`?

Reply:
The top level object should only have keys `url`, `options` and `variants`, so the `maxZoom` should not be in `L.TileLayer.Provider.providers.OpenPtMap` but in `L.TileLayer.Provider.providers.OpenPtMap.options`.